tsconfig.json

```diff
@@ -14,7 +14,6 @@
   },
   "include": [
     "src/**/*",
-    "test/**/*",
```

Comment:
this means.. we are not taking advantage of TS autocompletion/inline errors when editing tests?